.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint checks for timeexecution
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
just curious, why do we use the oldest supported version for linting?

Reply:
Just minimizing changes for now. Will change this one later.